src/caja-spatial-window.c

```diff
@@ -64,6 +64,7 @@
 
 #if !GTK_CHECK_VERSION(3,0,0)
 #define gtk_widget_get_preferred_size(x,y,z) gtk_widget_size_request(x,y)
+#define gtk_vbox_new(X,Y) gtk_box_new(GTK_ORIENTATION_VERTICAL,Y)
```

Comment:
this looks different to me :smile: 
https://github.com/mate-desktop/mate-screensaver/commit/dd837483d4435bfac418ef9c006cff642990a83f